src/dynamo0p3.py

```diff
@@ -2683,7 +2630,11 @@ def generate(self):
             # kernel (enforce_bc_kernel) arguments
             if self._kern.name.lower() == "enforce_bc_code" and \
                unique_fs.orig_name.lower() == "any_space_1":
```

Comment:
I don't know either. Maybe it was to help us trap any changes to the interface to `enforce_bc_code` that the Met Office didn't tell us about?